src/store/model/microcenter.ts

```diff
@@ -10,7 +10,7 @@ const microCenterLocationToId: Map<string, string> = new Map([
 	['chicago', '151'],
 	['columbus', '141'],
 	['dallas', '131'],
-	['devin', '181'],
```

Comment:
Lol... No idea. Sorry about that. Shout out to the CU Boulders :wink:

Reply:
lol it's all good, thanks for putting this whole thing together. Saw it as an easy way to boost my really limited github score.